gc/include/omrmm.hdf

```diff
@@ -203,6 +203,16 @@ typedef uintptr_t (*condYieldFromGCFunctionPtr) (OMR_VMThread *omrVMThread, uint
 		<struct>MM_InitializedEvent</struct>
 		<data type="struct OMR_VMThread*" name="currentThread" description="current thread" />
 		<data type="uint64_t" name="timestamp" description="time of event" />
+	</event>
+
+	<event>
+		<name>J9HOOK_MM_OMR_INITIALIZED_FULL</name>
```

Comment:
is this temporary (is there intention for this reverting back to plain J9HOOK_MM_OMR_INITIALIZED)?

if not, perhaps this could be called ..._V2 ?